.github/workflows/test.yaml

```diff
@@ -42,19 +42,19 @@ jobs:
                       dependencies: lowest
                       allowed-to-fail: false
                       variant: normal
-                    - php-version: '7.4'
+                    - php-version: '8.0'
                       dependencies: highest
                       allowed-to-fail: false
                       variant: 'symfony/symfony:"4.4.*"'
-                    - php-version: '7.4'
+                    - php-version: '8.0'
                       dependencies: highest
                       allowed-to-fail: false
                       variant: 'sonata-project/admin-bundle:"3.*"'
-                    - php-version: '7.4'
+                    - php-version: '8.0'
```

Comment:
maybe we should drop imagine 0.7?

Reply:
Yes this can be done in a minor release 👍🏻